edm4eic.yaml

```diff
@@ -244,6 +244,22 @@ datatypes:
   ## ==========================================================================
   ## Calorimetry
   ## ==========================================================================
+  edm4eic::SimCalorimeterPulse:
```

Comment:
We would really want to have generic structures to be able to consistently handle basic operations (adding noise, merging pulses, etc) for different detectors, not just calorimeters.

Would it be acceptable to make a more generic-sounding name, even if we keep the edm4hep::SimCalorimeterHit relation?

Reply:
See also a similar change pushed through in edm4hep for technical reasons outlined there; https://github.com/key4hep/EDM4hep/commit/cb738b0e2a5a8d4c922aa2a0d49974caf4adcd2e#diff-41291f568217a9755476ae21a12825127ae5a00179b19e867cffd64e3ceea36f

Reply:
I added a SimPulse interface type that can be used if this becomes needed in the future (then a SimTrackerPulse can also implement the SimPulse interface).